chia/full_node/full_node.py

```diff
@@ -2013,6 +2019,10 @@ async def peak_post_processing_2(
 
         self._state_changed("new_peak")
 
+        if ppp_result.signage_points is not None:
+            for signage_point in ppp_result.signage_points:
+                await self.signage_point_post_processing(*signage_point)
+
```

Comment:
I would have thought we'd want this early in this function, to minimize the reordering of these things